docs/source/usersguide/processing.rst

```diff
@@ -103,18 +103,119 @@ The code snippet below illustrates one possible configuration of parameters to t
 
 .. code-block:: python
 
-    import openmoc.process as proc
+    import openmoc.process
 
     # Retrieve the simulation state(s) stored in the 'states.h5' file
     # and returns the data in a Python dictionary
-    simulation_state = proc.restore_simulation_state(filename='states.h5')
+    simulation_state = openmoc.process.restore_simulation_state(filename='states.h5')
 
 
------------------------
-Computing Fission Rates
------------------------
+------------------------
+Computing Reaction Rates
+------------------------
 
-In some cases, a user may wish to only compute and export the fission rates in each flat source region for a simulation. In this case, the ``compute_fission_rates(...)`` routine in the ``openmoc.process`` module  may be used. The routine takes in a ``Solver`` subclass (e.g., ``CPUSolver``, ``VectorizedSolver``, ``GPUSolver``, etc.) and computes the fission rate for each ``Universe`` in the ``Geometry`` by summing up the fission rates in each ``Cell`` in the ``Universe``. In most cases, a ``Universe`` is replicated in many places throughout the ``Geometry``. To account for this, the routine will separately compute the fission rates for each unique placement of that ``Universe`` in the ``Geometry``. By default, the fission rates will be exported to a Python pickle_ file, but may alternatively be exported to an HDF5_ binary file. Each fission rate will be indexed by a string representing the "path" of ``Universes``, ``Lattices`` and ``Lattice`` cell indices traversed through the ``Geometry`` to reach the flat source region of interest. :ref:`Table 4 <table_fission_rates>` describes the parameters accepted by the routine.
+
+Reaction Rates By Tally Mesh
+----------------------------
+
+The ``openmoc.process`` module supports reaction rate through structured Cartesian mesh tallies, similar to how one might compute such quantities in a Monte Carlo code such as MCNP, Serpent or OpenMC. The ``Mesh`` class allows one to tally a reaction rate in each of its structured cells. :ref:`Table 4 <table_mesh_params>` below itemizes each of the property attributes of the ``Mesh`` class which can be used to tally reaction rates.
+
+.. _table_mesh_params:
+
+========================  ===================  ==========  ==========================================================
+Property                  Type                 Default     Note
+========================  ===================  ==========  ==========================================================
+``dimension``             list of integers     None        The number of mesh cells in each direction
+``lower_left``            list of floats       None        The lower-left corner of the structured mesh
+``upper_right``           list of floats       None        The upper-right corner of the structrued mesh
```

Comment:
Do we want to go with `lower_left` and `upper_right` or `origin` and `offset` as is done with a lattice?